packages/jammin-sdk/simulator.test.ts

```
@@ -153,3 +159,80 @@ describe("generateGuarantees", () => {
+    const jam = TestJam.empty();
+    expect(jam.state).toBeInstanceOf(SerializedState);
+  });
+});
+
+describe("TestJam.fromGenesis", () => {
+  // Serialise a JipChainSpec to the JIP-4 JSON shape (hex-encoded keys/values).
+  function toJip4Json(genesis: ReturnType<typeof generateGenesis>) {
+    return {
+      id: genesis.id,
+      bootnodes: genesis.bootnodes ?? [],
+      genesis_header: genesis.genesisHeader.toString().substring(2),
```

Biome (2.4.15)

[error] Avoid using substring and consider using slice instead.

(lint/style/noSubstr)